bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -33,6 +35,19 @@
 				<label>IP Address</label>
 				<description>Please add the IP Address of your sonnen battery.</description>
 			</parameter>
+			<parameter name="authToken" type="text">
+				<context>Authentication Token</context>
```

Comment:
This is not a valid context, see possible values here: https://github.com/openhab/openhab-core/blob/56fa0c9851175b93177c564be73f0088b5b9d5f6/bundles/org.openhab.core.config.core/src/main/java/org/openhab/core/config/core/ConfigDescriptionParameter.java#L315-L339